iceoryx_hoofs/source/cxx/filesystem.cpp

```diff
@@ -22,6 +22,7 @@ namespace iox
 {
 namespace cxx
 {
+// AXIVION Next Line AutosarC++19_03-M5.17.1 : corresponding assignment version of operator<< not required.
```

Comment:
@mossmaurice there were some talks about using the QAC++ syntax. Was the outcome to switch to the AXIVION syntax? What to do with the usage of the QAC++ suppression syntax in our current code base?